generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/execution/convertArgumentValue.kt

```diff
@@ -109,7 +109,11 @@ private fun <T : Any> mapToKotlinObject(input: Map<String, *>, targetClass: KCla
     // filter parameters that are actually in the input in order to rely on parameters default values
     // in target constructor
     val constructorParametersInInput = constructorParameters.filter { parameter ->
```

Comment:
we will probably need to rename this
we can just call it `constructorParameters`, and do

```kotlin
val constructorParameters = targetConstructor.parameters.filter { ... }
```

Reply:
Done